Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM php:7.4-alpine3.12
```

Comment:
A small discussion needs here.
As far as we still keep 5.4 as a minimum version in composer we have to check the compatibility during testing. It can be left as is(or as 7.0?) as far as we'll grow the major version of ApiBundle in upcoming changes and will change the composer restrictions consistently.